scss/_patterns_footer.scss

```diff
@@ -3,13 +3,15 @@
 // Footer pattern styling
 @mixin vf-p-footer {
   .p-footer {
-    border-top: 1px dotted $color-x-dark;
-    font-size: $sp-small;
+    border-top: 1px solid $color-mid-light;
+    color: $color-dark;
```

Comment:
I only added it as the spec is explicit - if we remove it here, we should remove it there @yaili?